### DISCUSSION TITLE
👑 [需求]关于proform和json标准

### DISCUSSION THREAD

**Discussion by phoema** [ant-design/pro-components#80] [Ideas]
### 🥰 需求描述

请问pro-form考虑支持json规范吗
目前阿里系我看有form-render 、formily等
是否考虑比如封装formily的antd方案，达到表单配置化？
就像antCharts封装g2plot一样？

**Comment by ritingliudd01** [comment]
同问

**Comment by crashsol** [comment]
formily 的解决方案确实大大提升了form的开发速速，很好用

**Comment by spirobel** [comment]
你们最喜欢formily 还是 form-render？
